pkg/cmd/gitprovider/delete.go

```diff
@@ -31,23 +32,31 @@ var gitProviderDeleteCmd = &cobra.Command{
 			log.Fatal(apiclient_util.HandleErrorResponse(res, err))
 		}
 
-		var gitProviderData apiclient.SetGitProviderConfig
-		gitProviderData.BaseApiUrl = new(string)
-		gitProviderData.Username = new(string)
-
 		if len(gitProviders) == 0 {
 			views.RenderInfoMessage("No git providers registered")
 			return
 		}
 
-		gitprovider_view.GitProviderSelectionView(&gitProviderData, gitProviders, true)
+		var gitProviderData apiclient.SetGitProviderConfig
+		gitProviderData.Id = new(string)
+		gitProviderData.BaseApiUrl = new(string)
+		gitProviderData.Username = new(string)
+		gitProviderData.Alias = new(string)
+
+		gitprovider_view.GitProviderDeleteView(&gitProviderData, gitProviders, apiClient, ctx)
 
-		if gitProviderData.Id == "" {
+		if gitProviderData.Id == nil {
 			log.Fatal("Git provider id can not be blank")
 			return
 		}
 
-		_, err = apiClient.GitProviderAPI.RemoveGitProvider(ctx, gitProviderData.Id).Execute()
+		for _, gp := range gitProviders {
```

Comment:
![Screenshot 2024-09-25 at 13 23 15](https://github.com/user-attachments/assets/4cd37835-2540-4231-90e2-52b5754927be)

The first line of the removal command is broken and unnecessary, please remove it.